Makefile

```diff
@@ -162,6 +162,15 @@ else
 
 endif
 
+# CMake 3.30+ removed support for cmake_minimum_required < 3.5.
+# Some external dependencies (klt_feature_tracker, jsoncpp) have old CMake files that
+# trigger errors on newer CMake versions. This flag tells CMake to accept older
+# minimum versions, allowing these dependencies to build without modification.
+CMAKE_VERSION := $(shell cmake --version 2>&1 | grep -oE "[0-9]+\.[0-9]+\.[0-9]+" | head -n1)
+ifeq ($(shell printf '%s\n3.27' "$(CMAKE_VERSION)" | sort -V | head -n1),3.27)
+    CMAKE_ARGS += -DCMAKE_POLICY_VERSION_MINIMUM=3.5
+endif
```

Comment:
What about updating these repos? I can help with it.

Reply:
The core issue is that the gazebo libraries include jsoncpp which has the < 3.5 version. I think the only way to fix it is if Gazebo updates their jsoncpp

Reply:
Oh, yea, we shouldn't open that can of worms I suppose.